crates/radix-engine-toolkit-uniffi/tests/bindings/example_test.go

```diff
@@ -199,4 +199,18 @@ func TestRetPublicKeyFingerprint(t *testing.T) {
 	assert(t, fingerprint.Bytes == "ABCDEFGHIJ", "Conversion of string failed")
 	assert(t, len(converted) == 10, "Length of byte array is invalid")
 	assert(t, reflect.DeepEqual(converted, bytes), "Conversion of byte array failed")
-}
\ No newline at end of file
+}
+
+func TestRetDecimalToFromBytes(t *testing.T) {
```

Comment:
It feels strange that we added a test for this in Go but not in the other languages. I think that I would prefer if the functionality is tested in Rust as a normal unit test instead of Go.

Reply:
I've added unit test to uniffi Decimal implementation.